{"uuid":"d439f278-b3a6-4ca1-83e7-c1c8669b644b","historyId":"7f1e2265f3a239b3f8c3b1ee0be5b0d2","testCaseId":"[engine:junit-jupiter]/[class:com.example.foodtourbackend.service.DishesServiceImplTest]/[method:addDishes_ShouldThrowNotFoundException_WhenRestaurantDoesNotExist()]","testCaseName":"addDishes_ShouldThrowNotFoundException_WhenRestaurantDoesNotExist()","fullName":"com.example.foodtourbackend.service.DishesServiceImplTest.addDishes_ShouldThrowNotFoundException_WhenRestaurantDoesNotExist","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:com.example.foodtourbackend.service.DishesServiceImplTest]/[method:addDishes_ShouldThrowNotFoundException_WhenRestaurantDoesNotExist()]"},{"name":"host","value":"DESKTOP-RQV9M0O"},{"name":"thread","value":"9292@DESKTOP-RQV9M0O.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"com.example.foodtourbackend.service.DishesServiceImplTest"},{"name":"testClass","value":"com.example.foodtourbackend.service.DishesServiceImplTest"},{"name":"testMethod","value":"addDishes_ShouldThrowNotFoundException_WhenRestaurantDoesNotExist"},{"name":"suite","value":"com.example.foodtourbackend.service.DishesServiceImplTest"}],"links":[],"name":"addDishes_ShouldThrowNotFoundException_WhenRestaurantDoesNotExist()","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\nAuthentication$MockitoMock$zsOPUYnH cannot be returned by EntityToDishesResponseDTO()\nEntityToDishesResponseDTO() should return DishesResponseDTO\n***\nIf you're unsure why you're getting above error read on.\nDue to the nature of the syntax above problem might occur because:\n1. This exception *might* occur in wrongly written multi-threaded tests.\n   Please refer to Mockito FAQ on limitations of concurrency testing.\n2. A spy is stubbed using when(spy.foo()).then() syntax. It is safer to stub spies - \n   - with doReturn|Throw() family of methods. More in javadocs for Mockito.spy() method.\n","trace":"org.mockito.exceptions.misusing.WrongTypeOfReturnValue: \nAuthentication$MockitoMock$zsOPUYnH cannot be returned by EntityToDishesResponseDTO()\nEntityToDishesResponseDTO() should return DishesResponseDTO\n***\nIf you're unsure why you're getting above error read on.\nDue to the nature of the syntax above problem might occur because:\n1. This exception *might* occur in wrongly written multi-threaded tests.\n   Please refer to Mockito FAQ on limitations of concurrency testing.\n2. A spy is stubbed using when(spy.foo()).then() syntax. It is safer to stub spies - \n   - with doReturn|Throw() family of methods. More in javadocs for Mockito.spy() method.\n\r\n\tat com.example.foodtourbackend.service.DishesServiceImplTest.addDishes_ShouldThrowNotFoundException_WhenRestaurantDoesNotExist(DishesServiceImplTest.java:98)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1746542999219,"stop":1746542999248}